.gemini/config.yaml

```
@@ -0,0 +1,2 @@
+code_review:
+  disable: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)